fetchai/ledger/serialisation/shardmask.py

```diff
@@ -9,7 +9,7 @@ class ShardMask:
     def state_to_address(cls, state, contract):
         # TODO: note circular dependency, as this will be called by contract
         assert contract.owner, "Contract does not have an owner"
-        return "{}.{}.state.{}".format(contract.digest.to_hex(), contract.owner, state)
+        return "{}.state.{}".format(contract.owner, state)
```

Comment:
`owner` should be replaced by `address`: 
```suggestion
        return "{}.state.{}".format(contract.address, state)
```